src/entities/shapeEntity.js

```diff
@@ -1,58 +1,62 @@
-function ShapeEntity(Interceptor, shapeObject, shapeArgs, canvasX, canvasY) {
-  const self = this;
-  /* global BaseEntity */
-  BaseEntity.call(self, shapeObject, shapeArgs, canvasX, canvasY);
-  this.areaAbs = 0;
-  this.type = Interceptor.currentColor + ` ` + shapeObject.name;
-  this.area = 0;
-  this.length = 0;
-
-
-  this.populate = function(shapeObject, arguments, canvasX, canvasY) {
-    this.location = this.getLocation(shapeObject, arguments, canvasX, canvasY);
-    this.coordLoc = this.canvasLocator(shapeObject, arguments, canvasX, canvasY);
-    if(!shapeObject.name.localeCompare(`ellipse`) || !shapeObject.name.localeCompare(`rect`)  || !shapeObject.name.localeCompare(`triangle`) || !shapeObject.name.localeCompare(`quad`)) {
-      this.areaAbs = this.getObjectArea(shapeObject.name, arguments);
-      this.area = (this.getObjectArea(shapeObject.name, arguments) * 100 / (canvasX * canvasY)).toFixed(2) + `%`;
-    } else if(!shapeObject.name.localeCompare(`line`)) {
```

Comment:
@spiray - looks like this is still not merged in..

Reply:
Fixed!